MySaver/MauiExceptions.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace MySaver;
+
+public static class MauiExceptions
+{
+#if WINDOWS
+    private static Exception _lastFirstChanceException;
+#endif
+
+    // We'll route all unhandled exceptions through this one event.
+    public static event UnhandledExceptionEventHandler UnhandledException;
+
+    static MauiExceptions()
+    {
+        // This is the normal event expected, and should still be used.
+        // It will fire for exceptions from iOS and Mac Catalyst,
+        // and for exceptions on background threads from WinUI 3.
+
+        AppDomain.CurrentDomain.UnhandledException += (sender, args) =>
+        {
+            UnhandledException?.Invoke(sender, args);
+        };
+
+#if IOS || MACCATALYST
```

Comment:
I guess we don't really need IOS or MAC support but it doesn't really hurt to have this.